charts/incubator/teamspeak/values.yaml

```
@@ -15,7 +15,7 @@ image:
 
 # -- environment variables. See more environment variables in the [teamspeak image documentation](https://hub.docker.com/_/teamspeak).
 # @default -- See below
-env:
+env: {}
   # TS3SERVER_LICENSE: accept
```

```suggestion
env:
  TZ: UTC
  # TS3SERVER_LICENSE: accept
```

Instead of specifying an empty dict, we tend to specify a default timezone var.